hztb-rest-api/src/main/java/com/svs/hztb/api/sm/model/refresh/OpinionData.java

```diff
@@ -7,50 +7,55 @@
 
 public class OpinionData {
 	private int opinionId;
-	//private int requestedUserId;
+	// private int requestedUserId;
 	private int requestedGroupId;
 	private String productName;
 	private Product product;
 	private Map<String, Integer> responseCounts = new HashMap<String, Integer>();
-	
+
 	public int getOpinionId() {
 		return opinionId;
 	}
+
 	public void setOpinionId(int opinionId) {
 		this.opinionId = opinionId;
 	}
-//	public int getRequestedUserId() {
-//		return requestedUserId;
-//	}
-//	public void setRequestedUserId(int requestedUserId) {
-//		this.requestedUserId = requestedUserId;
-//	}
+
+	// public int getRequestedUserId() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)